HouseRules_Essentials/Rules/StatusEffectConfigRule.cs

```diff
@@ -0,0 +1,69 @@
+﻿namespace HouseRules.Essentials.Rules
+{
+    using System;
+    using Boardgame;
+    using DataKeys;
+    using HarmonyLib;
+    using HouseRules.Types;
+
+    public sealed class StatusEffectConfigRule : Rule, IConfigWritable<global::Types.StatusEffectData[]>, IPatchable, IMultiplayerSafe
+    {
+        public override string Description => "Status Effect Durations are modified";
+
+        private static bool _isActivated;
+        private static global::Types.StatusEffectData[] _adjustments;
+
+        public StatusEffectConfigRule(global::Types.StatusEffectData[] adjustments)
```

Comment:
Generally, lists (e.g. `List<StatusEffectData>`) are preferred over arrays (See [this item from Effective Java](https://www.informit.com/articles/article.aspx?p=2861454&seqNum=3), although the same (mostly) applies to CSharp).

Lists will also un/marshal into JSON the same as arrays.

Reply:
Resolved in https://github.com/orendain/DemeoMods/pull/164/commits/bdfa98024eeffde696e67bff9cd2ac0250ee2bfb